RandLAPACK/drivers/rl_cqrrp.hh

```diff
@@ -57,9 +59,11 @@ class CQRRP_blocked : public CQRRPalg<T, RNG> {
             T ep,
             int64_t b_sz
         ) {
-            timing = time_subroutines;
-            eps = ep;
+            timing     = time_subroutines;
+            eps        = ep;
             block_size = b_sz;
+            use_qp3 = false;
+            use_gaussian = false;
```

Comment:
Setting use_gaussian to false meant fall back on sparse, right? I'd prefer to remove support for sparse sketching altogether in this context.

Reply:
Same comment as above.